### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by vjeux** [tailuge/billiards#7]
Hey @tailuge, I just wanted to say thank you for building this project. I’ve long been wanting to build an algorithm to suggest routes given the current set of balls. I found this project to implement the physics engine and debugging tools to be super convenient to play with my ideas.

Current status:
I’m able to place two balls randomly on the table and shot the other one in the pocket. Demo: https://vjeux.github.io/billiards/dist/diagrams.html

Next is to start building the route planning. I have an idea that I believe will work.

Some of the tweets from my hack session today:
- Capture and normalize table from my garage: https://twitter.com/vjeux/status/1337885313113358336?s=21
- Make a shot: https://twitter.com/vjeux/status/1337991561401884672?s=21
- Find the best hole: https://twitter.com/vjeux/status/1338009999239311361?s=21

So far it’s been dead simple to run and haven’t hit any bugs.
The only thing is that the spins don’t behave quite like the physical world. If I get far enough in my prototype I’ll give some more concrete repro, my hunch is that it’s going to be about tweaking the constants.

**Comment by tailuge** [comment]
Thanks for using my billiards project. 

I'm not used to typescript and I think the problems probably go deeper than just the constants. I hope to return to this project to improve the physics in the future.

It was great to see the pots going in on the demo page :)

**Comment by vjeux** [comment]
> It was great to see the pots going in on the demo page :)

Yeah, this is so satisfying. Spent 5 minutes just refreshing for no other reason :p
